doc/api/esm.md

```diff
@@ -585,8 +610,8 @@ format for that resolved URL given by the **ESM_FORMAT** routine.
 
 The _"module"_ format is returned for an ECMAScript Module, while the
 _"commonjs"_ format is used to indicate loading through the legacy
-CommonJS loader. Additional formats such as _"wasm"_ or _"addon"_ can be
-extended in future updates.
+CommonJS loader. Additional formats such as _"addon"_ can be extended in future
+updates.
```

Comment:
There's one additional mention of the non-existent `--entry-type` CLI flag 7 lines below this one. Is this the right PR to remove or update that material?